Makefile

```diff
@@ -133,13 +134,24 @@ install-linters: ## Install linters
 
 install-deps-libc: configure-build ## Install locally dependencies for testing libskycoin
 	git clone --recursive https://github.com/skycoin/Criterion $(BUILD_DIR)/usr/tmp/Criterion
-	mkdir $(BUILD_DIR)/usr/tmp/Criterion/build
+	mkdir -p $(BUILD_DIR)/usr/tmp/Criterion/build
```

Comment:
wasn't this removed already ? isn't it using `check` now ?

Reply:
That is already fixed in `simelo:stdevHan_t11_device_build_matrix`